### PR TITLE
Apply idle time check to event handler channels

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Apply idle time check to event handler channels (limit: 10 minutes)
- Apply life time check to event handler channels (limit: 1 hour) 
- Terminate websocket when there are errors in making connections
- Add getEventHandlerStatus() for monitoring 

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1290

### CODE REVIEW

File: event-handler/event-channel-manager.js [hunk 1]
```diff
@@ -61,18 +93,21 @@ class EventChannelManager {
...
     });
     this.startHeartbeat(this.wsServer);
+    this.startIdleCheck();
   }
 
   handleConnection(webSocket) {
     const LOG_HEADER = 'handleConnection';
     try {
       if (this.getNumEventChannels() >= NodeConfigs.MAX_NUM_EVENT_CHANNELS) {
+        webSocket.terminate();
```
Comment (jiyoung-an) [inline event-handler/event-channel-manager.js:103]: 👍

**Review by jiyoung-an — APPROVED** [review]
LGTM with some comments!
Thanks!

File: event-handler/event-channel-manager.js [hunk 2]
```diff
@@ -340,7 +377,27 @@ class EventChannelManager {
...
+  }
+
+  startIdleCheck() {
+    const LOG_HEADER = 'startIdleCheck';
+    this.idleCheckInterval = setInterval(() => {
+      for (const channel of Object.values(this.channels)) {
+        const idleTimeMs = channel.getIdleTimeMs();
+        if (idleTimeMs > NodeConfigs.EVENT_HANDLER_CHANNEL_IDLE_TIME_LIMIT_SECS * 1000) {
+          logger.info(`[${LOG_HEADER}] Closing long-idle channel: ${JSON.stringify(channel.toObject())}`);
+          channel.webSocket.terminate();
```
[the remark this answers is not in the document]
Comment (platfowner) [reply event-handler/event-channel-manager.js]: right. just removed. thanks!

File: event-handler/event-channel-manager.js [hunk 3]
```diff
@@ -340,7 +377,27 @@ class EventChannelManager {
...
+          this.closeChannel(channel);
+        }
+        const lifeTimeMs = channel.getLifeTimeMs();
+        if (lifeTimeMs > NodeConfigs.EVENT_HANDLER_CHANNEL_LIFE_TIME_LIMIT_SECS * 1000) {
+          logger.info(`[${LOG_HEADER}] Closing long-life channel: ${JSON.stringify(channel.toObject())}`);
+          channel.webSocket.terminate();
```
[the remark this answers is not in the document]
Comment (platfowner) [reply event-handler/event-channel-manager.js]: ditto~

File: event-handler/event-channel-manager.js
Comment (jiyoung-an) [inline event-handler/event-channel-manager.js:1]: 👍

**Comment by platfowner** [comment]
Thanks for the fast review! @jiyoung-an Now merging..

**Pull request merged**